### PR TITLE
Create ISSUE_TEMPLATE.md

### PR DESCRIPTION
**Pull request by fazerlicourice7**
This is an issue template for the Mifos X Web App.

### CODE REVIEW

**Comment by mbj36** [comment]
Thanks for the PR, It will be reviewed Shortly!

This message was posted by [GitMate.io](https://gitmate.io) with timestamp signature `Thanks for the PR, It will be reviewed Shortly!:1eQY1A:HFV1CUfjP9YQ0aLuUyBSzy8H1Eg`

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+## Description
+Provide a brief overview of what is not working.
```
Comment (ankit01ojha) [inline .github/ISSUE_TEMPLATE.md]: @fazerlicourice7 rather than writing ```what is not working``` it could be ```issue/enhancement```. As sometimes, people give enhancements also. I guess you will get my point. :)

Comment (ankit01ojha) [inline .github/ISSUE_TEMPLATE.md]: @fazerlicourice7 rather than writing ```what is not working``` it could be ```issue/enhancement```. :)

File: .github/ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+## Description
+Provide a brief overview of the enhancement or issue.
+
+## Steps to Reproduce
+Be as detailed as possible when providing steps to reproduce.
```
Comment (ManrajGrover) [inline .github/ISSUE_TEMPLATE.md]: Let's reword this to `Please add as much detail as possible to reproduce the error/issue`

File: .github/ISSUE_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+## Steps to Reproduce
+Be as detailed as possible when providing steps to reproduce. 
+Don't assume the reader knows anything about the Mifos X Web App.
+
+## Expected Behaviour
+
+## Actual Behaviour
+
+## Settings
+Mifos X version -
```
Comment (ManrajGrover) [inline .github/ISSUE_TEMPLATE.md]: Let's make this an unordered list

```
- Mifos X version:
- Browser used:
```

**Comment by luphieanza** [comment]
ah, thanks.

**Comment by luphieanza** [comment]
I'm a google code-in participant, in my gci dashboard the absence of the task, but in fact I have done it, do I get the value?
@mbj36 @ManrajGrover @ankit01ojha

**Comment by luphieanza** [comment]
[Guide how to use the issue tracker and how to report a good issue on Github.docx](https://github.com/openMF/community-app/files/1587011/Guide.how.to.use.the.issue.tracker.and.how.to.report.a.good.issue.on.Github.docx)
